lib/user_space/utrace.c

```diff
@@ -80,7 +84,11 @@ static int logbuf_map()
 		int available_bytes = sizeof trace_file_path -
 				      strlen(trace_file_path);
 		rc = snprintf(trace_file_path + strlen(trace_file_path),
-			available_bytes, "m0trace.%u", m0_pid_cached);
+			available_bytes,
+                        "m0trace.%u.%04d-%02d-%02d-%02d:%02d:%02d.%09"PRIu64,
+                        m0_pid_cached, tm.tm_year + 1900,
+                        tm.tm_mon + 1, tm.tm_mday, tm.tm_hour, tm.tm_min,
+                        tm.tm_sec, m0_time_nanoseconds(stamp));
```

Comment:
[suggestion]
I think we can remove nanoseconds else it will create long file name.

Reply:
Agree, nanoseconds can be dropped.

Reply:
incorporated